README.md

```diff
@@ -137,6 +137,13 @@ Within the proto files, the following types are supported:
   invocation) or between packages. All associations can be generated properly
   within the same package, but cross package only the belongs-to and many-to-many
   will work.
+- some repeated types can be automatically handled for Postgres by github.com/lib/pq, and
+  as long as the engine is set to postgres then to/from mappings will be created (see the
+  example called `postgres_arrays):
```

Comment:
You've got a dangling \` here. Also, I think it would be better to have a link, like on line 103 & 105 (although it appears those links are broken right now and should probably be `example/feature_demo/*.proto`).

Reply:
Alright, I changed this to provide the full path as a link to the postgres-arrays example (as this is the one using these arrays).